lib/promiscuous/kafka/file.rb

```diff
@@ -0,0 +1,78 @@
+class Promiscuous::Kafka::File
```

Comment:
this is for the file backend which is what powers the recorder. Best bet here is to have a generic Backend class instead of one per driver which is what it is now